.github/workflows/functional.yml

```diff
@@ -191,6 +191,7 @@ jobs:
 
       - name: Run the Offline Ceremony using RSTUF saving payload.json
         run: |
+          cp -r rstuf-cli/tests/files tests/
```

Comment:
Currently, I don't see where we access directly the `tests` folder.
Is it because the `rstuf -c rstuf.ini admin ceremony -b -u -f payload.json` which needs access to top-level `tests/`folder?